yw/Software/src/spellchecker/Read.java

```diff
@@ -0,0 +1,44 @@
+package spellchecker;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Read {
+
+	public List<String> read() throws IOException {
+		  String target_dir = "C:\\Users\\wangyifei\\software\\5030\\data";
+		  File dir = new File(target_dir);
+		  File[] files = dir.listFiles();
+		  List<String> myList = new ArrayList();
+		  
+		  for (File f : files) {
+			  if(f.isFile()) {
+				  BufferedReader inputStream = null;
+				  
+				  try {
+					  inputStream = new BufferedReader(new FileReader(f));
+					  String line;
+					  
+					  while ((line = inputStream.readLine())!=null) {
+						  myList.add(line);
```

Comment:
This line is adding all the stopwords of different languages to the same list and a non-stopword may be stripped only because it is spelled the same as a stopword of another language. Should we use a different data structure that divides stopwords of different languages (like a map or an array of lists)?

Reply:
This is a good suggestion, I will fix this problem as soon as possible